Source/Controllers/Other/SPSQLiteHistoryController.h

```diff
@@ -0,0 +1,34 @@
+//
+//  SPSQLiteHistoryController.h
+//  Sequel Ace
+//
+//  Created by James on 17/11/2020.
+//  Copyright © 2020 Sequel-Ace. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+#import <fmdb/FMDB.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface SPSQLiteHistoryController : NSObject
```

Comment:
why is it called `controller` if its's not type of `controller`? :)

Reply:
what would you suggest?

Reply:
`manager`?

Reply:
Done